extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -237,6 +258,11 @@ const chinaAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.azure.cn',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
+			azureLogAnalyticsResource: {
+				id: SettingIds.ala,
+				endpoint: 'https://api.loganalytics.io',
```

Comment:
And you've verified that this endpoint is correct for all these cloud environments? Typically there's different endpoints for each one.